src/align.cpp

```diff
@@ -851,7 +851,7 @@ static void align_same_func_call_params()
    deque<chunk_t *>  chunks;
    deque<AlignStack> as;
    AlignStack        fcn_as;
-   int               idx;
+   size_t            idx;
```

Comment:
http://www.cplusplus.com/reference/deque/deque/
wrote:
size_type	an unsigned integral type that can represent any non-negative value of difference_type	usually the same as size_t